docs/advtool-installation.html

```diff
@@ -64,50 +65,86 @@ <h4 class="ibm-h4">Taking advantage of leading edge tools and libraries</h4>
     <tr>
       <td>Debian 8</td>
       <td></td>
-      <td><p class="ibm-ind-link"><a class="ibm-anchor-down-link" href="#installUbuntu">Install Advance Toolchain</a></p><br>
-      <p class="ibm-ind-link"><a class="ibm-forward-link" href="https://developer.ibm.com/linuxonpower/advance-toolchain/advtool-compiler">Install cross compiler</a></p><br>
+      <td></td>
```

Comment:
do we need this extra empty cell?

Reply:
First cell for AT 12.0, second cell for AT 11.0 and third cell for AT 10.0.
As AT 10.0 is the only one supporting Debian 8, the first 2 cells have to stay empty.